lib/rules/web/admin_console/4.3/common_ui_elements.xyaml

```diff
@@ -343,7 +343,7 @@ check_key_and_resource_link:
 check_key_and_action_link:
   element: &key_action
     selector:
-      xpath: //dt[contains(text(),'<key>')]/following-sibling::dd[1]//button[text()='<text_in_action>'] | //button[(text()='<key>')]/parent::dt/following-sibling::dd[1]//button[text()='<text_in_action>']
+      xpath: //dt[contains(text(),'<key>')]/following-sibling::dd[1]//button | //button[(text()='<key>')]/parent::dt/following-sibling::dd[1]//button
```

Comment:
The text is not fixed, it will change as the total num of taints changing in my case.

Reply:
Does the change affect `click_key_and_action_link`? If no, I think we can also remove the `text_in_action` param from feature files.

```
 click_key_and_action_link:
     element:
        <<: *key_action
        op: click
```

Reply:
updated, thanks.